src/components/EmptyState/EmptyState.tsx

```diff
@@ -2,9 +2,9 @@ import React from 'react'
 import './emptyState.scss';
 import { ReactComponent as Progressing } from '../../assets/icons/ic-progressing.svg';
 
-function EmptyState({ children }) {
+function EmptyState({ children, className = '' }) {
     return (
-        <div className="flex column empty-state" style={{ width: '100%', height: '100%' }}>
+        <div className={`flex column empty-state ${className}`} style={{ width: '100%', height: '100%' }}>
```

Comment:
For `width: '100%', height: '100%'` can directly use shorthand class name

Reply:
done